### PR TITLE
chore: set default hardfork to `grayGlacier`; update `@ethereumjs/*` packages

### PR DESCRIPTION
**Pull request by davidmurdoch**
This changes the default hardfork to `grayGlacier` (was previously `london`). This change also updates `@ethereumjs/*` packages to their latest versions, which is what allows us to add the `grayGlacier` hardfork (`ganache --hardfork grayGlacier`). This hardfork does not change ganache behavior, other than make the hardfork name available.

### CODE REVIEW

File: src/chains/ethereum/ethereum/package.json [hunk 1]
```diff
@@ -72,8 +72,8 @@
     "env-paths": "2.2.1",
     "eth-sig-util": "2.5.3",
     "ethereumjs-abi": "0.6.8",
-    "ethereumjs-util": "7.1.4",
-    "keccak": "3.0.1",
+    "ethereumjs-util": "7.1.5",
+    "keccak": "3.0.2",
```
Comment (davidmurdoch) [inline src/chains/ethereum/ethereum/package.json:76]: `@ethereumjs/*` packages have updated to this version of `keccak`, so I matched our dependency with theirs so we don't have to install multiple.

File: src/chains/ethereum/block/package.json [hunk 2]
```diff
@@ -56,13 +56,13 @@
...
-    "@ethereumjs/common": "2.6.4",
+    "@ethereumjs/common": "2.6.5",
     "@ganache/ethereum-address": "0.5.0",
     "@ganache/ethereum-transaction": "0.5.0",
     "@ganache/ethereum-utils": "0.5.0",
     "@ganache/rlp": "0.5.0",
     "@ganache/utils": "0.5.0",
     "emittery": "0.10.0",
-    "ethereumjs-util": "7.1.4"
+    "ethereumjs-util": "7.1.5"
```
Comment (davidmurdoch) [inline src/chains/ethereum/block/package.json:66]: `@ethereumjs/*` packages have updated to this version of `ethereumjs-util`, so I matched our dependency with theirs so we don't have to install multiple.

File: src/chains/ethereum/ethereum/package.json [hunk 3]
```diff
@@ -97,7 +97,6 @@
     "@types/lodash.clonedeep": "4.5.6",
     "@types/lru-cache": "5.1.1",
     "@types/mocha": "9.0.0",
-    "@types/secp256k1": "4.0.1",
```
Comment (davidmurdoch) [inline src/chains/ethereum/ethereum/package.json:100]: Noticed this wasn't even used.

File: src/chains/ethereum/transaction/package.json [hunk 4]
```diff
@@ -48,17 +48,17 @@
...
+    "@ethereumjs/common": "2.6.5",
     "@ganache/ethereum-address": "0.5.0",
     "@ganache/ethereum-utils": "0.5.0",
     "@ganache/rlp": "0.5.0",
     "@ganache/secp256k1": "0.4.0",
     "@ganache/utils": "0.5.0",
-    "ethereumjs-util": "7.1.4",
-    "node-gyp-build": "4.2.3"
+    "ethereumjs-util": "7.1.5",
+    "node-gyp-build": "4.5.0"
```
Comment (davidmurdoch) [inline src/chains/ethereum/transaction/package.json:58]: `@ethereumjs/*` packages have updated to this version of `node-gyp-build`, so I matched our dependency with theirs so we don't have to install multiple.

File: src/chains/filecoin/filecoin/package.json [hunk 5]
```diff
@@ -73,8 +73,8 @@
     "@types/ws": "7.4.7",
     "async-sema": "3.1.0",
     "base32-encoding": "1.0.0",
-    "blakejs": "1.1.0",
-    "bn.js": "5.1.2",
+    "blakejs": "1.2.1",
+    "bn.js": "5.2.1",
```
Comment (davidmurdoch) [inline src/chains/filecoin/filecoin/package.json:77]: These versions were updated elsewhere, so I updated them here, too.

File: src/packages/rlp/package.json [hunk 6]
```diff
@@ -49,7 +49,7 @@
   ],
   "dependencies": {
     "@ganache/utils": "0.5.0",
-    "rlp": "2.2.6"
+    "rlp": "2.2.7"
```
Comment (davidmurdoch) [inline src/packages/rlp/package.json:52]: `@ethereumjs/*` packages have updated to this version of `rlp`, so I matched our dependency with theirs so we don't have to install multiple.

**Review by MicaiahReid — APPROVED** [review]
I'm fine with `chore`. Or `build(ethereumjs)` maybe? But maybe not since it's not just version updates but also adding `grayGlacier`

**Comment by jeffsmale90** [comment]
What about `feat(hardfork)` ?

**Review by jeffsmale90 — APPROVED** [review]
Lovin' it 🍟

**Pull request merged**